src/main/java/heronarts/lx/dmx/DmxModulator.java

```diff
@@ -26,7 +26,7 @@
 
 @LXModulator.Global("DMX")
```

Comment:
With a new DMX category, it seems like the UI name for this could just be "Channel", and we could drop the "DMX" prefix from the other modulator device names?

Reply:
For sure.  Seems like the names of this modulator and Dmx16bitModulator should match.  Like "1-Channel and 2-Channel" or "8-bit and 16-bit" or "Basic/Simple and High-Resolution".

The DMX prefix isn't needed in the Add Modulator menu when grouped by category, but should we leave it for the default modulator name?  Otherwise, a modulator of name "Channel" in the modulators instance list seems a little vague, could be confused with something related to a LX Mixer Channel.